### PR TITLE
Style add new art

### PR DESCRIPTION
**Pull request by jrpespisa**
Styling added for New and Edit Art forms.

We started working on the photos resource. It is incomplete, but we would like to deploy today so we are wrapping up this pull request

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -12,6 +12,6 @@
  *
  *= require_tree .
  *= require_self
+ *= require_new
```
Comment (cmkoller) [inline app/assets/stylesheets/application.scss]: You shouldn't need to add this new line to load the new file - `require_tree .` should load in all files in the stylesheets folder! (While you're at it, move `require foundation_and_overrides` to the top of this little `require` chunk to make sure your styles always take precedence over Foundation's!)

File: app/assets/stylesheets/new.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+$border-color:  #000000;
+
+.border {
+  border: 2px solid $border-color;
+  margin: auto;
+  padding: 20px 30px;
+}
+
+body {
+  // background-image: url(https://static.pexels.com/photos/6989/city-love-rainbow-nyc.jpg);
```
Comment (cmkoller) [inline app/assets/stylesheets/new.scss]: Get rid of commented-out code!

File: app/assets/stylesheets/new.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  background-size: cover;
```
Comment (cmkoller) [inline app/assets/stylesheets/new.scss]: You shouldn't need this rule about background-size if you don't have a background image!

File: app/controllers/photos_controller.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    @photo.art = @art
+
+    unless @photo.save
+      flash[:error] = @photo.errors.full_messages.join('. ')
+      render :new
+    end
+  end
+
+  def destroy
+  end
```
Comment (cmkoller) [inline app/controllers/photos_controller.rb]: Avoid having this action set up if you aren't using it yet

File: app/views/arts/_new_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+  <div class="field">
+    <%= f.label "Description:" %>
+    <%= f.text_field :description %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Add New Art", class: 'button small round form_button' %>
+  </div>
+<% end %>
```
Comment (cmkoller) [inline app/views/arts/_new_form.html.erb]: These two forms are pretty much exactly the same - can you combine them into the same partial? If you just do `<%= f.submit, class: "..." %>` without passing in the name of the submit button, it should automatically generate a pretty okay name for the button based on whether you're updating or creating.

**Comment by cmkoller** [comment]
There should be a test for uploading a photo! Right now I only see a test for creating a new `Art` object

File: app/uploaders/art_photo_uploader.rb [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# encoding: utf-8
+
```
Comment (EliseFitz15) [inline app/uploaders/art_photo_uploader.rb]: This file should be in the photo pr

Comment (EliseFitz15) [inline app/uploaders/art_photo_uploader.rb]: Just remove that file and the test image file and the gem in the gem file and then this will be all set.

**Pull request merged**